tests/manifests/test-object.yaml

```diff
@@ -18,5 +18,17 @@ spec:
   preservePoolsOnDelete: false
   gateway:
     port: 80
-    # securePort: 443
     instances: 1
+  security:
```

Comment:
Let's have this section in a dedicated file like `tests/manifests/test-kms-vault-spec.yaml` and append it to `tests/manifests/test-object.yaml` during the integration test.

Reply:
If I use dedicated session from `tests/manifests/test-kms-vault-spec.yaml`, then I end up adding/modifying following fields for `test-object.yaml` by calling sed/yq
```  
      VAULT_BACKEND: "v2"
        VAULT_SKIP_VERIFY: "false"
        VAULT_CACERT: vault-ca-cert
        VAULT_CLIENT_CERT: vault-client-cert
        VAULT_CLIENT_KEY: vault-client-key
```
To avoid this I have added entire block again in `test-object.yaml`